assets/javascript/app.js

```diff
@@ -5,35 +5,44 @@ var incorrectanswer;
 var unanswered;
 var userpicked;
 //  When the resume button gets clicked, execute the run function.
-$("#startbutton").on("click", run);
-
-function myFunction(optradio1) {
-
-    $("result1").value = optradio1;
-};
-
-
-//  Clearing the intervalId prior to setting our new intervalId will not allow multiple instances.
-function run() {
-    clearInterval(intervalId);
-    intervalId = setInterval(decrement, 1000);
-    console.log('intervalId', intervalId);
-}
-
-
-
-
-//  The decrement function.
-function decrement() {
-    //  Decrease number by one.
-    number--;
-    //  Show the number in the #show-number tag.
-    $("#timeout").text(number);
-    //  Once number hits zero...
-    if (number === 0) {
-        $("#questionnaire").hide();
+$(document).ready(function() {
+
+    $("#startbutton").on("click", run);
+    
+    $(".radioBtn").on('click', function(event) {
```

Comment:
I've added class='radioBtn' to each radio button.  When their clicked, we know which question it was because of the event.target.name.  And we know the value of the button clicked by event.target.value.